apps/projects/tasks.py

```diff
@@ -1,10 +1,16 @@
 import importlib
 
-from background_task import background
+from celery import shared_task
 
 
-@background(schedule=1)
-def send_async_no_object(email_module_name, email_class_name, object, args, kwargs):
-    mod = importlib.import_module(email_module_name)
-    cls = getattr(mod, email_class_name)
-    return cls().dispatch(object, *args, **kwargs)
+@shared_task
+def send_async_no_object(
```

Comment:
Lets pls rename it to something else.. send_email_data or anything other than this.

Reply:
I think the idea is to replace this one with the refactored one in a4, right?

Reply:
as discussed today we will postpone any real refactoring to another PR and focus on making celery work. I have there simply removed the return statement.